src/test/java/com/exasol/SelectIT.java

```diff
@@ -51,4 +52,28 @@ void testEmptySelectList() {
         assertRlsQueryWithUser("SELECT 'foo' FROM " + getVirtualSchemaName(sourceSchemaName) + ".T", user,
                 table().row("foo").row("foo").matches());
     }
+
+    @Test
+    void testSelectWithOrderByColumnAndLimit() {
+        final String sourceSchemaName = "ORDER_LIMIT_SCHEMA";
+        final Schema sourceSchema = createSchema(sourceSchemaName);
+        sourceSchema.createTable("T", "NR", "INTEGER").insert(1).insert(2).insert(3);
+        final VirtualSchema virtualSchema = createVirtualSchema(sourceSchema);
+        final User user = createUserWithVirtualSchemaAccess("ORDER_LIMIT_USER", virtualSchema);
+        assertRlsQueryWithUser("SELECT NR FROM " + getVirtualSchemaName(sourceSchemaName)
+                        + ".T ORDER BY NR LIMIT 2", user,
+                table().row(1).row(2).matches(TypeMatchMode.NO_JAVA_TYPE_CHECK));
+    }
+
+    @Test
+    void testSelectWithOrderByExpressionAndLimitWithOffset() {
+        final String sourceSchemaName = "ORDER_LIMIT_OFFSET_SCHEMA";
+        final Schema sourceSchema = createSchema(sourceSchemaName);
+        sourceSchema.createTable("T", "TXT", "VARCHAR(10)").insert("a").insert("bb").insert("ccc").insert("dddd");
+        final VirtualSchema virtualSchema = createVirtualSchema(sourceSchema);
+        final User user = createUserWithVirtualSchemaAccess("ORDER_LIMIT_OFFSET_USER", virtualSchema);
+        assertRlsQueryWithUser("SELECT TXT FROM " + getVirtualSchemaName(sourceSchemaName)
+                        + ".T ORDER BY LENGTH(TXT) LIMIT 2 OFFSET 1", user,
```

Comment:
```suggestion
                        + ".T ORDER BY LENGTH(TXT) ASC LIMIT 2 OFFSET 1", user,
```
❓

Reply:
The renderer only adds `ASC` if the VS sets that switch. And it doesn't. This is equivalent and keeps the statement more compact.